### PR TITLE
WIP: Use staticfiles for front-end dependencies instead of CDNJS

### PR DESCRIPTION
**Pull request by acdha**
* [x] Install django-npm
* [x] Replace cdnjs references with {% static %} calls
* [ ] Tune include list to only have the dist files we care about

I'm not entirely in love with this approach but it avoids the support burden of dealing with Webpack. [`@pika/web`](https://github.com/pikapkg/web) is intriguing but not everything we are using supports ESM.

### CODE REVIEW

File: concordia/settings_template.py [hunk 1]
```diff
@@ -36,10 +36,43 @@
...
+    "openseadragon": ["build/*"],
+    "codemirror": ["lib/*", "addon/*", "mode/*"],
+    "prettier": ["*.js"],
+    "remarkable": ["dist/*"],
+    "jquery": ["dist/*"],
+    "js-cookie": ["src/*"],
+    "popper.js": ["dist/*"],
+    "bootstrap": ["dist/*"],
+    "screenfull": ["dist/*"],
+    "@fortawesome/fontawesome-free/": [
```
Comment (rstorey) [inline concordia/settings_template.py]: I think you meant fontawesome here, not fortawesome

File: package.json [hunk 2]
```diff
@@ -16,10 +16,19 @@
         "stylelint-config-recommended": "^3.0.0"
     },
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.8.2",
```
Comment (rstorey) [inline package.json]: Fortawesome or fontawesome?

Comment (acdha) [reply package.json]: FortAwesome is the organization: https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers#installing-free

Comment (rstorey) [reply package.json]: Oh I was not aware of that.

**Pull request closed without merging**